e2e-tests/playwright/tests/functional/channels/drafts/drafts_on_deleted_message.spec.ts

```diff
@@ -52,7 +52,7 @@ test('MM-T5435_1 Global Drafts link in sidebar should be hidden when another use
     await sidebarRight.postMessage(draftMessageByUser);
 
     // # Close the RHS for draft to be saved
-    await sidebarRight.close()
+    await sidebarRight.close();
```

Comment:
hmm i see we have currently no way to catch these 🤔

Reply:
I see. Will address separately.